### DISCUSSION TITLE
Local notification doesn't show up on ios 16.2

### DISCUSSION THREAD

**Discussion by Brionvega** [thudugala/Plugin.LocalNotification#345] [Q&A] [answered]
Hi, I can't manage to get notifications on latest iOS 16.2.
It seems that CustomUserNotificationCenterDelegate never triggers.

> config.AddiOS(iOS =>
> 						{
> 
> 							iOS.SetCustomUserNotificationCenterDelegate(new CustomUserNotificationCenterDelegate());
> 							iOS.SetPermission(new iOSNotificationPermission
> 							{
> 								LocationAuthorization = iOSLocationAuthorization.Always
> 							});
> 
> 						});

Any solution or advice?

Thanks,

Ben

**Comment by thudugala** [comment, marked as answer]
@Brionvega Try version 10.1.1

**Comment by fverjan** [comment]
Where should I Add this code to get permissions ask on IOs?

**Reply by thudugala** [reply]
@fverjan Call INotificationService RequestNotificationPermission

```cs
if (await _notificationService.AreNotificationsEnabled() == false)
{
    await _notificationService.RequestNotificationPermission();
}

var ff = await _notificationService.Show(request);
```
